### ISSUE TITLE
Tailwind JIT do not generate class with slash after it

### ISSUE THREAD

**Issue by Foprta** [tailwindlabs/tailwindcss#6358]
**What version of Tailwind CSS are you using?**

v2.2, v3

**What build tool (or framework if it abstracts the build tool) are you using?**

angular: 12.2

**What version of Node.js are you using?**

v16.13.0

**What browser are you using?**

Chrome

**What operating system are you using?**

Windows, Linux

**Reproduction URL**

https://github.com/Foprta/tailwind-repr

```
npm ci
npm run build:lib
npm run start
```

Text with `text-blue-500` is not blue

**Describe your issue**

Compiled angular library in node_modules have code:

![image](https://user-images.githubusercontent.com/26553030/145348141-91a95ff9-e826-446b-9562-4efd71ccbf37.png)

Our purge paths are : `['./node_modules/our-angular-lib/**/*.js']`

And it does not generate classes with a slash in the end, like highligthed `tw-bg-gray-400`.

If i create one more `tw-bg-gray-400` right before the existing one, JIT will generate it.

tailwindcss: 2.2.19

**Comment by ortuagustin** [comment]
I have the same issue with tailwind 3; setup is similar to OP, a vue library that ships some shared components. Classes ending in slash are not picked by JIT engine, ie:

```js
// somewhere in a large js built package...

\"class\": \"grid md:grid-cols-12\"\n    }
```

Class `grid` is picked by JIT engine; `md:grid-cols-12` is not picked. Will try changing JIT regex or disable minification/whitespace handling on the library

**Comment by linoberlin** [comment]
Could be fixed by changing the class-candidate regex in the expandTailwindAtRules.js file

from
_/([^<>"'`\s]*[^"'`\s:])/.source_
to
_/([^<>"'`\s]*[^"'`\s:\\])/.source_

ref: https://github.com/tailwindlabs/tailwindcss/commit/553fbaae6ced64440322a8cb76913cc868333614
_works fine in my testing_

**Comment by thecrypticace** [comment]
Fixed in #6590
